tests/tests/test_api_internal.py

```diff
@@ -13,62 +13,72 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 import os
-from common import internal_client, management_client, mongo, clean_db, inventory_attributes
+from common import (
```

Comment:
ok, it is hard to tell here, since lots of formatting has changed, what exactly was modified here? the API should stay the same.

Reply:
The only thing I've changed here is the logic for validating inventory attributes by filtering the returned attributes by scope == "inventory" (L105:106).

Reply:
looks good then